test/unit/helpers/api/integrations_helper_test.rb

```diff
@@ -230,4 +230,16 @@ class IsHttpTest < self
       refute(is_https?(nil))
     end
   end
+
+  class ProxyRulesPreview < self
```

Comment:
That will reexecute all tests of the parent class ...

Reply:
Fixed